libnext/backend.py

```diff
@@ -136,7 +140,11 @@ def __init__(self) -> None:
         self.backend.destroy()
         self.display.destroy()
 
-    # Properties
+    # Cleanup on SIGNINT
+    def cleanup(self) -> None:
```

Comment:
cleanup logic is in the init function rn as display.run is a blocking call, I will split it into a different function